### ISSUE TITLE
Split file based documents are not being printed correctly.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [fleszar1/flexpaper#222]
```
What steps will reproduce the problem?
1. Use the viewer to load a document using the split file loader method rather 
than a SWF based document.
2. Print the document using any of the available options from either the print 
button or the print menu.

What is the expected output? What do you see instead?
Expect to see a print job with the expected number of pages based on the print 
options chosen by the user in the job queue. Currently the job has "N/A" pages.

What version of the product are you using? On what operating system?
1.4.5 RC 1 and 1.4.5 on Windows XP.

Please provide any additional information below.

```

Original issue reported on code.google.com by `hayden.g...@gmail.com` on 5 Oct 2011 at 3:18

**Comment by GoogleCodeExporter** [comment]
Original comment by `erik.eng...@devaldi.com` on 2 Nov 2011 at 2:41
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
There seems to be different delays in timeout on different browsers, we are 
still looking into if we can extend this delay somewhat, are you given an error 
when you are trying? 
```

Original comment by `erik.eng...@devaldi.com` on 11 Nov 2011 at 3:35

**Comment by GoogleCodeExporter** [comment]
```
I think this was the issue that we fixed with the patch I emailed to you. The 
printSplitPaper method refers to a Movie type, but in the split mode the event 
target is a Bitmap.
```

Original comment by `hayden.g...@gmail.com` on 11 Nov 2011 at 3:39

**Comment by GoogleCodeExporter** [comment]
```
Fixed in version 1.5.0
```

Original comment by `erik.eng...@devaldi.com` on 2 Feb 2012 at 4:01
- Changed state: **Fixed**
